docs/content/stable/reference/configuration/yb-tserver.md

```diff
@@ -403,6 +403,12 @@ Assigns an extra priority to automatic (minor) compactions when automatic tablet
 
 Default: `50`
 
+##### --ysql_colocate_database_by_default
+
+When enabled, all databases created in the cluster are colocated by default. Must be set when the cluster is created. For more details, see [clusters in colocated tables](../../../architecture/docdb-sharding/colocated-tables/#clusters).
```

Comment:
The flag can be changed after the universe creation. But it requires process restarts.

Reply:
@hari90 - when you say process restarts , it is only the yb-tserver restart or yb-master and any other process as well?